src/ir/types/schema.h

```diff
@@ -22,6 +23,40 @@ class Schema {
 
   friend class TypeFactory;
 
+  //return a string representing the Schema in the following format: 
+  // name: <name or empty>
+  // fields: 
+  //    <field_name>: <field_kind>
+  //... 
+  //only showing the first layer of the schema, e.g. not returning the values of <field_kind>
+  //for easier testing, fields are sorted alphabetically based on their name  
+  std::string ToString(){
+    
+    std::string stringValue = absl::StrCat("name: ", name_.value_or(""), "\n", "fields: \n"); 
```

Comment:
`stringValue` does not follow style guide: https://google.github.io/styleguide/cppguide.html#General_Naming_Rules

Also, use a more meaningful name than `string_value`?